docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -122,6 +122,43 @@ Elliptic Curve Signature Algorithms
         :returns: A new instance of a :class:`EllipticCurvePublicKey`
             provider.
 
+Elliptic Curve Key Exchange algorithm
+-------------------------------------
+
+.. class:: ECDH(algorithm)
```

Comment:
In the case of SSH, a KDF is used, but not directly on the shared key returned by the key exchange operation. Part of the reason for this is that several keys are generated, and so this shared key is hashed with other pieces of information to generate a unique key stream for each of the keys. That means the key exchange needs to return the raw key value to use in multiple KDF calls.

Personally, I'd rather see the KDF removed entirely here. In the case where you only need a single KDF the caller can always apply it on the key value that's returned, and that will give them more flexibility about how it is applied, allowing for things like the SSH use case where multiple KDFs are needed.